install_config/registry/extended_registry_configuration.adoc

```diff
@@ -362,23 +362,31 @@ middleware:
       options:
         acceptschema2: false <2>
         pullthrough: true <3>
-        enforcequota: false <4>
-        projectcachettl: 1m <5>
-        blobrepositorycachettl: 10m <6>
+        mirrorpullthrough: true <4>
+        enforcequota: false <5>
+        projectcachettl: 1m <6>
+        blobrepositorycachettl: 10m <7>
   storage:
     - name: openshift <1>
 ----
-<1> These entries are mandatory. Their presence ensures required components get loaded. These values shouldn't be changed.
+<1> These entries are mandatory. Their presence ensures required components get
+loaded. These values shouldn't be changed.
 <2> Allow to store
 link:https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#image-manifest-version-2-schema-2[manifest
-schema v2] during a push to the registry. See xref:middleware-repository-acceptschema2[below] for more details.
-<3> Let the registry act as a proxy for remote blobs. See xref:middleware-repository-pullthrough[below] for more details.
-<4> Prevent blob uploads exceeding size limit defined in targeted project.
-<5> An expiration timeout for limits cached in the registry. The lower the
+schema v2] during a push to the registry. See
+xref:middleware-repository-acceptschema2[below] for more details.
+<3> Let the registry act as a proxy for remote blobs. See
+xref:middleware-repository-pullthrough[below] for more details.
+<4> Let the registry cache blobs served from remote registries for later fast
+access. The mirroring starts when the blob is accessed for the first time. The
+option has no effect if the xref:middleware-repository-pullthrough[pullthrough]
+is disabled.
+<5> Prevent blob uploads exceeding size limit defined in targeted project.
```

Comment:
s/Prevent/Prevents
s/exceeding size/exceeding the size
s/in targeted/in the targeted